crates/sui-sdk/src/apis.rs

```diff
@@ -389,6 +739,25 @@ impl CoinReadApi {
         Ok(coins)
     }
 
+    /// Return the balance of coins for the provided address, or an error upon failure.
```

Comment:
Needs more detail: Explaining that this function sums up the balances of all coins, and what happens if no filter is provided (does it produce a result that contains multiple balance outputs? I think not, I think it gives the balance of `Coin<SUI>`)